### PR TITLE
meme: init at 2017-09-10

### PR DESCRIPTION
**Pull request by fgaz**
###### Motivation for this change

An example of a meme generated with this program

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by fgaz** [comment]
I hope a name as short as this doesn't clash with something more important in the future

File: pkgs/applications/graphics/meme/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    rev = "a6521f2eecb0aac22937b0013747ed9cb40b81ea";
+    sha256 = "1gbsv1d58ck6mj89q31s5b0ppw51ab76yqgz39jgwqnkidvzdfly";
+  };
+
+  meta = with stdenv.lib; {
+    description = "A command line utility for creating image macro style memes";
+    homepage = "https://github.com/nomad-software/meme";
+    license = licenses.mit;
+    maintainers = [ maintainers.fgaz ];
+    platforms = platforms.linux;
```
Comment (grahamc) [inline pkgs/applications/graphics/meme/default.nix]: Upstream says this should work on Mac. Have you tested to see if this builds on mac via Nixpkgs?

**Comment by grahamc** [comment]
@GrahamcOfBorg build meme

File: pkgs/applications/graphics/meme/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/applications/graphics/meme/default.nix]: I don't have a mac to test it with. Should I set it to darwin too and try to trigger a build like you did?

Comment (fgaz) [reply pkgs/applications/graphics/meme/default.nix]: I... can't figure out how to do it. `with platforms; [ linux darwin ]` doesn't work. `[ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]` does, but strings are bad, right?

**Comment by fgaz** [comment]
@GrahamcOfBorg build meme

**Comment by fpletz** [comment]
@GrahamcOfBorg build meme

**Pull request merged**